app/ts/components/App.tsx

```diff
@@ -39,7 +39,7 @@ function fetchPayloadFromStorage() {
 
 function fetchSettingsFromStorage() {
 	// @TODO: add ability to manage settings
-	return { blocksInFuture: 2n, priorityFee: 10n ** 9n * 3n, relayEndpoint: MEV_RELAY_MAINNET }
+	return { blocksInFuture: 3n, priorityFee: 10n ** 9n * 3n, relayEndpoint: MEV_RELAY_MAINNET }
```

Comment:
is 3 more reliable? Or what is the idea behind making this so far in future?

Reply:
Should be, not sure it matters too much. I changed it testing UI on Goerli since it takes forever to get bundles included there.
This will be able to be changed in a settings UI in future.